scvi/inference/posterior_utils.py

```diff
@@ -1,13 +1,102 @@
+import os
 from typing import List, Optional, Union
 
+import anndata
 import numpy as np
 import pandas as pd
 import scipy
 import torch
+import torch.nn as nn
 from matplotlib import pyplot as plt
 from scipy.optimize import linear_sum_assignment
-from scipy.stats import kde, entropy
-from sklearn.neighbors import NearestNeighbors, KNeighborsRegressor
+from scipy.stats import entropy, kde
+from sklearn.neighbors import KNeighborsRegressor, NearestNeighbors
+
+from scvi.dataset import AnnDatasetFromAnnData
+
+
+def load_posterior(
+    dir_path: str,
+    model: nn.Module,
+    use_cuda: Optional[Union[bool, str]] = "auto",
```

Comment:
what does "auto" do? It's not being tested. We should also specify what it does in the docstring.